website/docs/markdown-features.mdx

```diff
@@ -932,3 +932,96 @@ class HelloWorld {
 You may want to implement your own `<MultiLanguageCode />` abstraction if you find the above approach too verbose. We might just implement one in future for convenience.
 
 If you have multiple of these multi-language code tabs, and you want to sync the selection across the tab instances, refer to the [Syncing tab choices section](#syncing-tab-choices).
+
```

Comment:
Great documentation! This will be really nice.

Reply:
thanks :)